doc/config-import.rst

```diff
@@ -491,6 +491,28 @@ in a structure of your choice, like:
 
 etc.
 
+**Attributes:**
+
+    ::
+
+        entry-separator="..."
+
+    * Optional
+    * Default: **"; "**
+
+    This string is used to join multi-valued items (e.g. Composer, Performer) into one string.
+
+    ::
+
+        legacy-separator="..."
```

Comment:
I just wonder if those separators could be per file rather than global...
Won't it be more straight-forward to leave up to js script to handle them? Only if js string can take \0.

Reply:
Changed